### PR TITLE
Remove the unwanted email icon

### LINKED ISSUE #151: Remove the uwanted email icons in the contact section
Remove this unwanted email icon

image

### PR DESCRIPTION
**Pull request by maddy020**
## Purpose

The purpose of this Pull Request is to fix #151<issue-number>

## Goals
One icon tag is used extra in the html file which needs to be removed

## Approach
One icon tag is used extra in the html file which needs to be removed

### Before

Before

### After
after

### Preview Link

https://pr-152-NEW-YOSHITHA-WEBSITE.surge.sh/dark

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: dark/index.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!doctYPE html>
```
Comment (YoshithaRathnayake) [inline dark/index.html:1]: Can you undo these changes? @maddy020 
```suggestion
<!DOCTYPE html>
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html [hunk 2]
```diff
@@ -180,6 +180,7 @@ <h1>
       </div>
       <div id="tooltip">
         <span id="tooltiptext">GitLab</span>
+	B
```
Comment (YoshithaRathnayake) [inline dark/index.html:183]: Can you tell me why did you add these changes? @maddy020

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html [hunk 3]
```diff
@@ -307,6 +308,6 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
       sidebar.classList.toggle("is-active");
     });
   </script>
-</body>
+</body>OOA
 
-</html>
+6[6~[[6~OOB</html>
```
Comment (YoshithaRathnayake) [inline dark/index.html:313]: Can you tell me why did you add these changes? @maddy020

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by maddy020** [comment]
I think it is due to system error 
I am changing this 
I am so sorry about that issue

File: dark/index.html (hunk 3 above)
Comment (maddy020) [reply dark/index.html]: I think It is a system error 
So sorry about that I have changed the file

**Comment by YoshithaRathnayake** [comment]
> I think it is due to system error I am changing this I am so sorry about that issue

No problem

File: dark/index.html [hunk 4]
```diff
@@ -180,6 +180,7 @@ <h1>
       </div>
       <div id="tooltip">
         <span id="tooltiptext">GitLab</span>
+
```
Comment (YoshithaRathnayake) [inline dark/index.html:183]: Can you remove these unwanted lines? @maddy020 
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Review by YoshithaRathnayake — APPROVED** [review]
Thank You! @maddy020
